Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -1,8 +1,8 @@
 {
     "id": "497D490F-EEA7-4F2B-AB94-48D9C1ACDCB1",
     "name": "AzureRmWebAppDeployment",
-    "friendlyName": "AzureRM App Service Deployment",
-    "description": "Deploy the web project using AzureRM App Service",
+    "friendlyName": "Azure App Service Deployment: ARM",
+    "description": "Update Azure App Service using Web Deploy / Kudu REST APIs",
```

Comment:
Kudu by default can also be interpreted as monitoring source control and deployment when anything changes to it. Kudu REST API was very explicit on what we are doing. @RoopeshNair ?